site/content/api/Command-And-Control-API.md

```diff
@@ -57,22 +53,23 @@ After execution of a control message on a device, device MUST send a response me
 
 **Preconditions**
 
-1. Client has established an AMQP connection with Hono.
-2. Client has established an AMQP link in role *sender* with Hono using target address `control-reply/${tenant_id}/{$device_id}`. Where `${device_id}` is an ID of the
+1. *Protocol Adapter* and *Business Application* have established an AMQP connection with the AMQP 1.0 Network.
```

Comment:
as stated above, I have strong doubts whether we actually want to define this implementation strategy as being the API.